app/modules/notifications/models.py

```diff
@@ -129,11 +129,13 @@ class NotificationChannel(str, enum.Enum):
         'email_template_name': 'collaboration_manager_create',  # Not yet written
         'email_digest_content_template': 'collaboration_manager_create_digest',
         'mandatory_fields': {'collaboration_guid', 'user1_name', 'user2_name'},
+        'allow_multiple': True,
     },
     NotificationType.collab_manager_revoke: {
         'email_template_name': 'collaboration_manger_revoke',  # Not yet written
         'email_digest_content_template': 'collaboration_manager_revoke_digest',
         'mandatory_fields': {'collaboration_guid', 'user1_name', 'user2_name'},
+        'allow_multiple': True,
```

Comment:
@naknomum Have a look at the merge notifications. Do you need to update them to allow multiple merge requests from the same user for different individuals?
Hint, I think you do.

Reply:
indeed it seems like i do.   created [DEX-730](https://wildme.atlassian.net/browse/DEX-730) for this, assigned to me.